.github/workflows/cross-merge.yml

```diff
@@ -0,0 +1,79 @@
+name: Cross merge
+
+on:
+  workflow_call:
+    secrets:
+      robot-token:
+        description: 'A PAT token used by the app/robot'
+        required: true
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+
+    steps:
+        - uses: actions/checkout@v2
+          with:
+            fetch-depth: 0
+            token: ${{ secrets.robot-token }}
+
+        - name: Inject slug/short variables
+          uses: rlespinasse/github-slug-action@v3.x
+
+        - name: Get merge map
+          uses: octokit/request-action@v2.x
+          id: map
+          # Allow failures to be ignored.
+          # We will consider output as empty
+          continue-on-error: true
```

Comment:
From what I see next steps depend on this one. We should not continue unless there is some execution path that I don't see at first glance.

Reply:
This is *also* used to avoid failures in the destination repository, as there will be no map available.

And then we check if there is any output in `destinations` step.

> And btw, doesn't it open PR's on each push, without checking that a PR already exists?

Yes, it does. This is why we ignore errors in the next step.